lambda/shared/src/main/scala/feral/lambda/LambdaEnv.scala

```diff
@@ -42,6 +42,15 @@ sealed trait LambdaEnv[F[_], Event] { outer =>
 object LambdaEnv {
   def apply[F[_], A](implicit env: LambdaEnv[F, A]): LambdaEnv[F, A] = env
 
+  def of[F[_]: Applicative, Event](e: Event, c: Context[F]): LambdaEnv[F, Event] =
+    new LambdaEnv[F, Event] {
+      val event = e.pure
+      val context = c.pure
+    }
+
+  def event[F[_], Event](implicit env: LambdaEnv[F, Event]): F[Event] = env.event
+  def context[F[_], Event](implicit env: LambdaEnv[F, Event]): F[Context[F]] = env.context
```

Comment:
Some convenience methods. Didn't do it here, but might less us improve the UI a bit.

```scala
// before
def myMethod[F[_]: Monad](implicit env: LambdaEnv[F, MyEvent]) = for {
  event <- env.event
} yield ???

// after
// we can provide these aliases for each type of lambda
type MyEventLambdaEnv[F[_]] = LambdaEnv[F, MyEvent]
def myMethod[F[_]: Monad: MyEventLambdaEnv] = for {
  event <- LambdaEnv.event
} yield ???
```

Subject to personal preference, of course. But I think I prefer the latter style.

Reply:
Did it. I do like it :)